M2/Macaulay2/packages/Complexes/ChainComplexDoc.m2

```diff
@@ -4363,6 +4372,66 @@ doc ///
         freeResolution
 ///
 
+doc ///
+    Key
+        (Ext,Module,Module)
+        (Ext,Ideal,Ideal)
+        (Ext,Ideal,Module)
+        (Ext,Ideal,Ring)
+        (Ext,Module,Ideal)
+        (Ext,Module,Ring)
+    Headline
+        total Ext module
+    Usage
+        Ext(M, N)
+    Inputs
+        M:Module
+            or ofClass{Ideal,Ring}, that is homogeneous
+        N:Module
+            or ofClass{Ideal,Ring} over the same ring as $M$, that is also homogeneous
+    Outputs
+        :Module
+            the $\operatorname{Ext}$ module of $M$ and $N$, as a
+            multigraded module, with the modules
+            $\operatorname{Ext}^i(M,N)$ for all values of $i$
+            appearing simultaneously.
+    Description
+        Text
+            The computation of the total Ext module is possible for modules over the
+            ring $R$ of a complete intersection, according to the algorithm
+            of Shamash-Eisenbud-Avramov-Buchweitz.  The result is provided as a finitely
+            presented module over a new ring with one additional variable of degree
+            $\{-2,-d\}$ for each equation of degree $d$ defining $R$.  The 
+            variables in this new ring have degree length $1$ more than the degree length of 
+            the original ring.  In other words, it is multigraded with the
+            degree $d$ part of $\operatorname{Ext}^n(M,N)$ appearing as the degree
+	        $\{-n,d\}$ part of $\operatorname{Ext}(M,N)$.
+        Text
+            We illustrate this in the following example.
+        Example
+            R = QQ[x,y]/(x^3,y^2);
+            N = cokernel matrix {{x^2, x*y}}
+            H = Ext(N,N);
+            ring H
+            S = ring H;
+            H
+            isHomogeneous H
+            rank source basis( { -2,-3 }, H)
+            rank source basis( { -3 }, Ext^2(N,N) )
+            rank source basis( { -4,-5 }, H)
+            rank source basis( { -5 }, Ext^4(N,N) )
+            hilbertSeries H
+            hilbertSeries(H,Order=>11)
+        Text
```

Comment:
You can also use the `References` keyword instead of `Text` (and undo indentation by one level).

Reply:
In this particular case, I think we like having the reference in place where it is,